### DISCUSSION TITLE
Viewing a file from a fork from within a PR fails

### DISCUSSION THREAD

**Discussion by sebastiaanspeck** [community/community#86074] [Mobile]
### Select Topic Area

Bug

### Body

When trying to view a file on a branch in a fork, from within a PR, fails if the file does not exist in the upstream repo. It shows that the file doesn’t exist.

This is due to the fact that the URL the app is trying to visit, is the path from the upstream repo instead of the fork itself.
![IMG_2006](https://github.com/community/community/assets/12570668/8d5fb608-c4a2-4537-b975-c97932fb6a5b)

**How to reproduce**

1. Create a new repo
2. Fork the repo
3. Create a new file in the fork
4. Open a PR
5. Visit the PR on GitHub Mobile
6. Open the Files view
7. Click on the three dots and choose View File
8. When clicking on Open in Browser, you will see that the URL is incorrect and could also not be opened in the browser.

**Comment by elisealix22** [comment]
Thank you for the detailed test steps @sebastiaanspeck ! I created a test repository to help us reproduce and opened an issue internally. We will look into it!

**Reply by sebastiaanspeck** [reply]
Is there any update on this?

**Comment by istudyatuni** [comment]
I should note that it worked at first, but after some app update it broke (a few months ago)

**Comment by officalcom** [comment]
hallo word
